odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1193,15 +1200,16 @@ def _validate_tag_img(self, node, name_manager, node_info):
         if not any(node.get(alt) for alt in self._att_list('alt')):
             src = next((node.get(src) for src in self._att_list('src') if node.get(src)), "")
             if src:
+                # VFE FIXME this str construction looks strange
                 src = ' with src (%s)'
-            msg = _('<img> tag%s must contain an alt attribute')
-            self.handle_view_error(msg % src, raise_exception=False)
+            self.handle_view_error(
+                '<img> tag%s must contain an alt attribute' % src, raise_exception=False)
```

Comment:
I think there is a missing space in the source too

Reply:
Not when `src` is empty ^^.